build: show tzdata version in tarantool module (#10387)
tarantool/tarantool: changelogs/unreleased/show-tzdata-version.md

> ## bugfix/build

Reviewer: Looks more like a feature.

Author: Fixed to `feature/lua`, thanks.

> * Introduced an option `tarantool.build.tzdata_version` that shows
>   tzdata version.

Reviewer 2: ```suggestion
* Introduced the `tarantool.build.tzdata_version` option to get 
the `tzdata` version.
```

Author: Updated, thanks!
